README.md

```diff
@@ -52,7 +52,7 @@ This repository provides tutorial code in C++ for deep learning researchers to l
 
 1. [C++-17](http://www.cplusplus.com/doc/tutorial/introduction/) compatible compiler
 2. [CMake](https://cmake.org/download/) (minimum version 3.14)
-3. [LibTorch version >= 1.12.0 and <= 1.13.1](https://pytorch.org/cppdocs/installing.html)
+3. [LibTorch version >= 1.12.0 and <= 2.0.0](https://pytorch.org/cppdocs/installing.html)
```

Comment:
Looks like we need to have?
```suggestion
3. [LibTorch version >= 2.0.0](https://pytorch.org/cppdocs/installing.html)
```

Reply:
`>= 1.12.0 and <= 2.0.0` is correct as (since the upgrade to 1.13.1) we support a range of libtorch versions. I am keeping the minimum version - currently 1.12.0 - unless there are backward incompatible changes introduced in the newer libtorch version or we ourselves use features that are not available in the older version. The benefit is that the CMake script won´t download the latest version if a supported version is already present. E.g. if you pull the 2.0.0 release but still have v1.13.1 downloaded (in `pytorch-cpp/libtorch` or pointed to using `CMAKE_PREFIX_PATH`) which is within the supported range, we will not force downloading of the latest version.